test/verify/check-metrics

```diff
@@ -796,6 +796,11 @@ class TestCurrentMetrics(MachineCase):
             with b.wait_timeout(300):
                 b.wait_in_text("#current-metrics-card-cpu", f"pod {shortid}")
             b.wait(lambda: topServiceValue(self, "Top 5 CPU services", "%", 1) > 70)
+
+            # It takes one re-render for the name lookup
+            with b.wait_timeout(300):
```

Comment:
... but that sholdn't take 5 minutes surely? This is updated every 5 seconds, isn't it?

Reply:
Yup, so let's reduce it.